### PR TITLE
Fix info icon missing on mobile

### PR DESCRIPTION
**Pull request by lubej**

### CODE REVIEW

**Comment by lubej** [comment]
> This is making it more difficult to maintain and breaks things on landscape phone
> 
> Before	After
> before	after

Ok. Reverting the height modifications. So only info icon is addressed.

**Comment by lukaw3d** [comment]
Still can't reach the footer on landscape mode

**Comment by lukaw3d** [comment]
Oh, you intentionally hid the footer on mobile?

**Review by lukaw3d — APPROVED** [review]

File: src/app/pages/HomePage/index.tsx [hunk 1]
```diff
@@ -156,15 +161,14 @@ export const HomePage: FC = () => {
...
 
-        <FooterStyled>
-          {showInfoScreenBtn && (
-            <IconButton aria-label={infoAriaLabel} onClick={onToggleInfoScreenClick}>
-              <InfoOutlinedIcon fontSize="medium" sx={{ color: 'white' }} />
-            </IconButton>
-          )}
-          {!isMobile && <Footer />}
-        </FooterStyled>
+        <FooterStyled>{!isMobile && <Footer />}</FooterStyled>
```
Comment (lukaw3d) [inline src/app/pages/HomePage/index.tsx:164]: I would simplify to `{!isTablet && <FooterStyled><Footer /></FooterStyled>` and remove breakpoints from FooterStyled

**Comment by lubej** [comment]
> Oh, you intentionally hid the footer on mobile?

Right, this is expected behaviour.

**Pull request merged**